vercel.json

```diff
@@ -2,12 +2,12 @@
   "redirects": [
     {
       "source": "/releases/mainnet/active/contracts.json",
-      "destination": "/releases/mainnet/1.9.11/contracts.json",
+      "destination": "/releases/mainnet/1.9.12/contracts.json",
```

Comment:
I think we should wait the execution of the spell before updating active links

Reply:
Yes if, this gets merged before the execution of the spell, then the active link should remain to 1.9.11